platform/kube/controller.go

```diff
@@ -284,30 +286,56 @@ func (c *Controller) Services() []*model.Service {
 	return out
 }
 
+// GetService implements a service catalog operation
+func (c *Controller) GetService(name string, namespace string) (*model.Service, bool) {
```

Comment:
There is only one service for all service tags, no? v1 and v2 still belong to the same service?